### ISSUE TITLE
Problem with compilation

### ISSUE THREAD

**Issue by Pretous** [faeibson/ESPPWMFanControl#3]
Hello, when I try to compile it got errors like below:

In file included from c:\users\michal\appdata\local\arduino15\packages\esp8266\tools\xtensa-lx106-elf-gcc\3.0.0-newlib4.0.0-gnu23-48f7b08\xtensa-lx106-elf\include\assert.h:10,
                 from c:\users\michal\appdata\local\arduino15\packages\esp8266\tools\xtensa-lx106-elf-gcc\3.0.0-newlib4.0.0-gnu23-48f7b08\xtensa-lx106-elf\include\sys\reent.h:503,
                 from c:\users\michal\appdata\local\arduino15\packages\esp8266\tools\xtensa-lx106-elf-gcc\3.0.0-newlib4.0.0-gnu23-48f7b08\xtensa-lx106-elf\include\stdlib.h:18,
                 from c:\users\michal\appdata\local\arduino15\packages\esp8266\tools\xtensa-lx106-elf-gcc\3.0.0-newlib4.0.0-gnu23-48f7b08\xtensa-lx106-elf\include\c++\10.2.0\cstdlib:75,
                 from c:\users\michal\appdata\local\arduino15\packages\esp8266\tools\xtensa-lx106-elf-gcc\3.0.0-newlib4.0.0-gnu23-48f7b08\xtensa-lx106-elf\include\c++\10.2.0\stdlib.h:36,
                 from C:\Users\Michal\AppData\Local\Arduino15\packages\esp8266\hardware\esp8266\3.0.0\cores\esp8266/Arduino.h:27,
                 from sketch\ESPPWMFanControl.ino.cpp:1:
C:\Users\Michal\Documents\Arduino\libraries\WhareHauoraWiFiManager/WiFiManager.h:25:24: error: 'const char HTTP_HEAD []' redeclared as different kind of entity
   25 | const char HTTP_HEAD[] PROGMEM            = "<!DOCTYPE html><html lang=\"en\"><head><meta name=\"viewport\" content=\"width=device-width, initial-scale=1, user-scalable=no\"/><title>{v}</title>";
      |                        ^~~~~~~
In file included from sketch\ESPPWMFanControl.h:14,
                 from D:\Pobrane\ESPPWMFanControl-1.0-beta2\ESPPWMFanControl\ESPPWMFanControl.ino:9:
C:\Users\Michal\AppData\Local\Arduino15\packages\esp8266\hardware\esp8266\3.0.0\libraries\ESP8266WebServer\src/ESP8266WebServer.h:47:39: note: previous declaration 'HTTPMethod HTTP_HEAD'
   47 | enum HTTPMethod { HTTP_ANY, HTTP_GET, HTTP_HEAD, HTTP_POST, HTTP_PUT, HTTP_PATCH, HTTP_DELETE, HTTP_OPTIONS };
      |                                       ^~~~~~~~~
Multiple libraries were found for "WiFiUdp.h"
 Used: C:\Users\Michal\AppData\Local\Arduino15\packages\esp8266\hardware\esp8266\3.0.0\libraries\ESP8266WiFi
 Not used: C:\Program Files (x86)\Arduino\libraries\WiFi
exit status 1
Error compiling for board LOLIN(WEMOS) D1 R2 & mini.

The error come even with clean install of arduino IDE

**Comment by faeibson** [comment]
Hi,
are you sure you got the right WifiManager library? Your WifiManager.h is located within the directory "WhareHauoraWifiManager"... Via Google I can find out that this repo seems to be a quite outdated fork of tzapu's WifiManager which is the one you should be using actually. 
So I think you just took the wrong library, uninstall it and use "WifiManager" by tzapu and you should be good!

Let me know if you could resolve the issue.

**Comment by Pretous** [comment]
Yes, I used correct library - to compile it I had to change HTTP_HEAD to HTTP_HEADS in library and it goes but now I got problem with WiFi connection - every reset of esp it lost every settings.

**Comment by betmensk** [comment]
unfortunately this project died ...
cannot be compiled, constant errors in libraries and when the compilation succeeds, the access point refuses to connect ...

**Comment by faeibson** [comment]
I will be looking into it when I have more free time. At the moment I'm quite busy unfortunately. I will keep you updated here.

**Comment by burnbabyburn** [comment]
The above errors seem to been fixed. ty
But brzo seems to have updated brzo_i2c to version 1.3.3 -which is buggy https://github.com/pasko-zh/brzo_i2c/issues/44
`In function 'brzo_i2c_write':`
`brzo_i2c.c:72:2: error: cannot find a register in class 'RL_REGS' while reloading 'asm'
   72 |  asm volatile (`

If one downgrade to 1.3.2 it compiles
